doc/src/tutorial/codegen.rst

```diff
@@ -98,27 +108,23 @@ anything. A use case for ``Piecewise``::
              tau = x
           end if
 
-The various printers also tend to support ``Indexed`` objects well.
-
-With ``contract=True`` these expressions will be turned into loops, whereas
+The various printers also tend to support ``Indexed`` objects well. With
+``contract=True`` these expressions will be turned into loops, whereas
 ``contract=False`` will just print the assignment expression that should be
 looped over::
 
-    >>> from sympy import Eq, IndexedBase, Idx
-    >>> from sympy import jscode
     >>> len_y = 5
-    >>> y = IndexedBase('y', shape=(len_y,))
-    >>> t = IndexedBase('t', shape=(len_y,))
+    >>> y_ = IndexedBase('y', shape=(len_y,))
+    >>> t_ = IndexedBase('t', shape=(len_y,))
```

Comment:
It'd be nicer if, instead of trailing underscores, we chose non-conflicting variable names.